apps/query-eval/queryeval/driver.py

```diff
@@ -255,12 +255,30 @@ def do_eval(self, _query: QueryEvalQuery, result: QueryEvalResult) -> QueryEvalR
             console.print("[yellow]:point_right: Dry run: skipping eval")
             return result
 
+        # Evalute query plans
+        if not _query.expected_plan:
+            console.print("[yellow]:construction: No expected query plan found, skipping.. ")
+        elif not result.plan:
+            console.print("[yellow]:construction: No computed query plan found, skipping.. ")
+        else:
+            plan_diff = _query.expected_plan.compare(result.plan)
+            if len(plan_diff) == 0:
+                console.print("[green]\U00002714 Plan correct")
```

Comment:
You wild man

(Python supports unicode directly in string literals so you can just use that :-)

Reply:
Fine, what's the fun in that though